scripts/vulnerabilities.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+#
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script prints a table of fixable vulnerabilities of medium severity or
+# higher, based on the latest or specified git tag.
+#
+# USAGE: $ scripts/vulnerabilities.sh [IMAGE_TAG]
+#
+# The IMAGE_TAG is optional. If not specified, it defaults to the latest tag of
+# the current local environment.
+#
+# Requires gcloud authn/authz to read the results of vulnerability scans from
+# Google Container Registry in the "config-management-release" project.
+# Required access:
+# - Storage Object Viewer on "artifacts.config-management-release.appspot.com"
+# - Container Registry Service Agent on Container Registry API
+#   (containerregistry.googleapis.com)
+# - Container Analysis Service Agent on Container Analysis API
+#   (containerscanning.googleapis.com)
+
+set -euo pipefail
+
+configsync_latest_image_tag="$(git describe --tags --abbrev=0)"
+configsync_image_tag="${1:-${configsync_latest_image_tag}}"
+
+# TODO: share with Makefile to keep in sync
```

Comment:
How would that work for the dependencies? They all need to be in the same project, but the dependencies are all hard-coded in the manifests, not dynamic like the ones from the makefile.

Reply:
I guess the dependencies don't technically need to be in the same project if they both have scanning enabled, but I'd have to make the project configurable, not just the whole image prefix, since the project needs to be passed to the gcloud command.

Reply:
Ok, I made the project configurable with GCP_PROJECT